yellowbrick/contrib/classifier/boundaries.py

```diff
@@ -512,14 +510,14 @@ def fit_draw(self, X, y=None, **kwargs):
         self.fit(X, y, **kwargs)
         self.draw(X, y, **kwargs)
 
-    def fit_draw_poof(self, X, y=None, **kwargs):
+    def fit_draw_show(self, X, y=None, **kwargs):
         """
         Fits a transformer to X and y then returns
         visualization of features or fitted model.
-        Then calls poof to finalize.
+        Then calls show to finalize.
         """
         self.fit_draw(X, y, **kwargs)
-        self.poof(**kwargs)
+        return self.show(**kwargs)
```

Comment:
Did you mean to return self.show() here?

Reply:
Yes, `show()` now returns `ax` to ensure notebooks work correctly, so we need to make sure we return it unless the visualizer is being returned.